keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -881,6 +887,74 @@ Other components:
 * check how the change of meaning of resource requests influence other
   Kubernetes components.
 
+### Instrumentation
+
+The kubelet will record the following metrics:
+
+#### `kubelet_container_resize_attempts_total`
+
+This metric tracks the total number of resize attempts observed by the Kubelet, counted at the container level.
+A single pod update changing multiple containers will be considered separate resize attempts.
+
+Labels: 
+- `resource` - what resource. Possible values: `cpu`, or `memory`. If more than one of these resource types is changing in the resize request, we increment the counter multiple times, once for each.
+- `type` - what type of resource is being resized. Possible values: `limits`, or `requests`. If more than one of these resource types is changing in the resize request, we increment the counter multiple times, once for each.
```

Comment:
what do we call this? `type` seems too vague...

Reply:
maybe `requirement`?

Reply:
thaaaaaank you :)